articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/machine-learning-and-deep-learning/
+title: Machine Learning and Deep Learning
+description:
+author:
+date: 2020-10-19T00:00:00-07:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+
+  - url: /engineering-education/machine-learning-and-deep-learning/hero.jpg
+    alt:
+---
+Technology is growing at a fast pace in recent years. Due to this, it is now easy to get lost in the rumpus of technological terms that keep popping up every day.
+<!--more-->
+### Introduction
+Machine learning and deep learning are among the fast-growing applications. Artificial Intelligence ([AI](https://www.investopedia.com/terms/a/artificial-intelligence-ai.asp)) is one of the fast-growing technologies that has contributed significantly to the growth of machine and deep learning.
+
+The AI incorporates and avails intelligence that is used to enhance decision making on both applications. These applications have contributed significantly to solving problems that previously affected performances in some applications.  
+
+New ideas in the tech world have led to various innovations that have sparked the nature of things. Alexa and Siri are [virtual assistant](https://www.investopedia.com/terms/v/virtual-assistant.asp) that enhance the use of machine learning and are widely used.
+
+Machine learning and deep learning have attracted significant attention over the years. The popularity of these two has led to various applications implementing its tools.
+
+This article will focus on understanding what machine learning and deep learning are. The reader will have a deeper understanding of how these applications differ and enhance their functionalities.
+
+### Machine Learning and Artificial Intelligence
+
+Machine learning is a [subset](https://en.wikipedia.org/wiki/Subset) of AI that allows computer algorithms to make accurate data outcomes through interpretations.
+
+Machine learning is used to obtain and use data that is used to make final decisions. Machine learning processes are also dependent on the subsets of artificial intelligence.
```

Comment:
The first sentence is wrong. What do you mean by ml is used to obtain and use data?